src/pattern-library/components/Library.tsx

```diff
@@ -255,11 +337,7 @@ function Demo({
             </div>
           </div>
         )}
-        {visibleTab === 'source' && (
-          <div className="border w-full rounded-md bg-slate-7 text-color-text-inverted p-4">
-            <ul>{source}</ul>
-          </div>
-        )}
+        {visibleTab === 'source' && source}
```

Comment:
I noticed there is a slight visual flicker when switching between demo and code, both in Chrome and Safari. Locally I found that I could eliminate this by rendering both the demo and the source tabs but hiding the non-visible tab using `display: 'none'`.